eav/models.py

```diff
@@ -156,6 +156,7 @@ class Attribute(models.Model):
                  change it's datatype.
     '''
 
+    @python_2_unicode_compatible
```

Comment:
the class Mate has no mthod __str__, maybe should add "@python_2_unicode_compatible" at its father class Attribute, this works for me